frontend/app/openproject-app.js

```diff
@@ -28,9 +28,12 @@
 
 var I18n = require('./vendor/i18n');
 
-// standard locales
-I18n.translations.en = require("locales/js-en.yml").en;
-I18n.translations.de = require("locales/js-de.yml").de;
+// load all js locales
+var requireExtraLocale = require.context('../../config/locales', false, /js-[\w|-]{2,5}\.yml$/);
```

Comment:
Just about naming stuff. I'd probably call this `localeFiles`.

More specific I would drop the term *extra*, because now that this is the default way of loading locales it does not add the "extra" locales, but *all* of them.
I am not sure if the term `require` served any purpose, at least I don't understand it (besides the fact that the variable is filled using a method called require).